client/src/components/Search.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react'
+import Axios from 'axios'
+
+const Search = () => {
+    const [searchTerm, setSearchTerm] = useState('')
+    const [novels, setNovels] = useState([])
+    useEffect(() => {
+        fetchNovels()
+    }, [])
+
+    const fetchNovels = async () => {
+        const { data } = await Axios.get('http://localhost:1337/api/novels')
+        setNovels(data)
+    }
+
+    return (
+        <div className="novels">
+            <div className="name">Great Novels</div>
+            <input
+                className="searchBox"
+                type="text"
+                name="search"
+                value={ searchTerm }
+                onChange={event => setSearchTerm(event.target.value)}
+            />
+            {
+                novels.filter(novels => novels.title.toLowerCase().includes(searchTerm.toLowerCase())).map(novel => {
```

Comment:
filter is heavy... should only run once when the searchTerm changes... not on each render.

look into useMemo